test/ToastNotificationTests/APITests.cpp

```diff
@@ -578,5 +578,14 @@ namespace Test::ToastNotifications
         {
             RunTestUnpackaged(L"VerifyToastUpdateZeroSequenceFail_Unpackaged", testWaitTime());
         }
+
+        TEST_METHOD(VerifyForegroundRegistrationSuccess)
+        {
+            RunTest(L"VerifyForegroundRegistrationSuccess", testWaitTime());
```

Comment:
What about the unpackaged scenario tests?

Reply:
Unpackaged/Packaged have the same behavior. But I can add an unpackaged test.